src/components/Infra/InfraEnvWizard.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react'
+import { InfraWizard } from 'openshift-assisted-ui-lib';
+import { useK8sModel, k8sCreate, history } from '@openshift-console/dynamic-plugin-sdk/api';
+import { Base64 } from 'js-base64';
+
+import { InfraEnvKind } from '../../kind';
+
+import 'openshift-assisted-ui-lib/dist/index.css';
+
+const InfraEnvWizard: React.FC = () => {
+  const [infraModel] = useK8sModel(InfraEnvKind);
+  const [secretModel] = useK8sModel('core~v1~Secret');
+  const [clusterDepModel] = useK8sModel('hive.openshift.io~v1~ClusterDeployment');
+  const [agentClusterInstallModel] = useK8sModel('extensions.hive.openshift.io~v1beta1~AgentClusterInstall');
+  const [name, setName] = React.useState<string>();
+  const onSubmit = React.useCallback(async (values) => {
+    setName(values.name);
+    await k8sCreate(secretModel, {
+      kind: 'Secret',
+      apiVersion: 'v1',
+      metadata: {
+        name: values.name,
+        namespace: 'assisted-installer'
+      },
+      data: {
+        '.dockerconfigjson': Base64.encode(values.pullSecret),
```

Comment:
Why don't we use simple `atob` or `btoa`?

Reply:
no reason, `btoa` it is!